wolfcrypt/src/ecc.c

```diff
@@ -6217,6 +6230,8 @@ int wc_ecc_verify_hash_ex(mp_int *r, mp_int *s, const byte* hash,
    CRYS_ECDSA_VerifyUserContext_t sigCtxTemp;
    word32 msgLenInBytes = hashlen;
    CRYS_ECPKI_HASH_OpMode_t hash_mode;
+#elif defined(WOLFSSL_SILABS_SE_ACCEL)
+   byte sigRS[ECC_MAX_CRYPTO_HW_SIZE * 4];
```

Comment:
I am surprised this is required to be 32 * 4. Each R and S elements should be 32 bytes each. Can you double check this. If its required please add comment as to why.

Reply:
Yep, not sure why I had increased it.